openapi/openapi.yaml

```
@@ -601,6 +604,7 @@ x-tagGroups:
       - Blocklists
       - KYC documents
       - AML
+      - Risk Score
```

issue: sentence case
```suggestion
      - Risk score
```